### DISCUSSION TITLE
[ChatBees.ai] Add an AI assistant on your website in seconds

### DISCUSSION THREAD

**Discussion by jon-snowman** [vercel/community#5719] [Show & Tell]
## tldr
We've built a plugin for Netlify that automatically adds an AI assistant to any website. We're in the process of porting it to vercel and would love some early feedback from the community

## Demo 
[Original website](https://65bbd0916a9bd103ba1694dc--transcendent-quokka-d0500e.netlify.app/) VS [website with plugin enabled](https://transcendent-quokka-d0500e.netlify.app/)

## About ChatBees

Hello! We're the co-founders of ChatBees (https://www.chatbees.ai/)

We've built a [simple-to-use plugin for Netlify](https://github.com/ChatBees/netlify-plugin-chatbees) that automatically adds an AI assistant on your website. We're in the process of porting it over to Vercel, but would like gather some early feedback from the community. Is this something you'd find useful? Is there any feature you'd like see?

-- Jun, Ryan and Kamal
